THESIS.md

```diff
@@ -64,10 +66,14 @@ From Razzaq et al.'s work, we extracted and classified a list of measurable DevX
 
```

Comment:
in this last paragraph, we should remove text about table, or refer to it - saying it is available in results section of our research.